LINK/etc/profile.d/evm.sh

```diff
@@ -13,4 +13,6 @@ function tailmiq() #If no value is given with tailmiq it defaults to the evm.log
 alias tailpglog='tail -f $APPLIANCE_PG_DATA/pg_log/postgresql.log'
 
 # Appliance Status:
-alias apstatus='echo "EVM Status:";service evmserverd status;echo " ";echo "HTTP Status:";service httpd status'
+alias apstatus='echo;systemctl status evmserverd -n 0;echo; \
+systemctl status httpd.service -n 0;echo;systemctl status memcached -n 0; \
+echo -e "\nPostgreSQL service:"; /opt/rh/$APPLIANCE_PG_SCL_NAME/root/usr/bin/pg_isready'
```

Comment:
My thinking was that systemctl will tell you if the database is running but not necessarily that it is accepting connections.  The other part is I would prefer that the entire output of apstatus would be short enough to fit on one screen. 

I'm open to suggestions, and want to find a good way of limiting the output of systemctl so we can provide the useful bits to show the application status without wasting screen real estate.